ntoskrnl/mm/section.c

```diff
@@ -3886,7 +3884,7 @@ NtQuerySection(
             }
             else
             {
-                DPRINT1("Unimplemented code path\n");
+                Sbi.BaseAddress = Section->Segment->BasedAddress;
```

Comment:
Note that this fallback case (taken in case all the others above fail), overwrites the default value `Sbi.BaseAddress = (PVOID)Section->Address.StartingVpn;` set at the top.
Is it OK? Was that default value sane, or should this be changed?
Is the default `Sbi.Size` value that is set at the top (and that you _**haven't**_ changed there) still OK?